### DISCUSSION TITLE
配置通过通配符采集日志，实际没有采集到

### DISCUSSION THREAD

**Discussion by hanzoi** [alibaba/ilogtail#496] [Help]
版本为预发布版本，k8s部署

以下为配置：
![image](https://user-images.githubusercontent.com/61114924/199178881-99fb57ae-5c44-41e8-a06f-d813619cc093.png)

需要采集的日志如下：
![image](https://user-images.githubusercontent.com/61114924/199178915-d1d37b7c-9358-48ca-a576-2964dd75e278.png)

目前有9个采集配置，只有这个使用了通配符的无法采集到，想确认下原因

**Comment by messixukejia** [comment]
实时产生的才能采集，历史数据需要用其他手段。

**Comment by yyuuttaaoo** [comment]
要判断是不是通配符的问题，可以将通配符替换成非通配符试一下。这个采集配置中有processor和kafka，可以去掉processor、使用stdout输出简化一下便于排查。还有问题可以，附上完整deployment yaml和采集yaml。
